phing/tasks/tests.xml

```diff
@@ -83,21 +83,45 @@
   <target name="tests:phpunit" description="Executes PHPUnit tests against custom code.">
     <delete dir="${reports.localDir}/phpunit" quiet="true"/>
     <mkdir dir="${reports.localDir}/phpunit"/>
+    <if>
+      <and>
+        <isset property="phpunit.local" />
+        <not><isset property="env.TRAVIS" /></not>
```

Comment:
I'd prefer not to add any Travis specific logic to our tasks. Pipelines will replace Travis in the future.